app/routes.py

```diff
@@ -1,2 +1,114 @@
-from flask import Blueprint
+from app import db
+from flask import Blueprint, jsonify, request,make_response
+from app.models.planet import Planet
 
+solarsystem_bp = Blueprint("solarsystem", __name__, url_prefix="/solarsystem")
+
+
+@solarsystem_bp.route("", methods=["GET"])
+def handle_solarsystem():
+    requested_name = request.args.get('name')
+    shorter_orbital_period=request.args.get("shorter_orbital_period")
+    if requested_name:
+        requested_name = requested_name.capitalize()
+        planets=Planet.query.filter_by(name=requested_name)
+    elif shorter_orbital_period:
+        try:
+            shorter_orbital_period = int(shorter_orbital_period)
+        except ValueError:
+            return {"Error": "Orbital period must be numeric"}, 400
+        planets=Planet.query.filter(Planet.orbital_period < request.args.get("shorter/orbital_period"))
+    elif request.args.get("order_by") == "name":
+        planets = Planet.query.order_by(Planet.name.desc())
+    else:
+        planets = Planet.query.all()
+    solarsystem_response = [] 
+    for planet in planets:
+        solarsystem_response.append(planet.to_dict())
+        
+    return jsonify(solarsystem_response) 
```

Comment:
For readability I suggest using response codes, even for 200s

```suggestion
    return jsonify(solarsystem_response) , 200
```